### ISSUE TITLE
Time format decoding

### ISSUE THREAD

**Issue by suvajit** [estiens/world_cup_json#75]
All match start time is returned as : 
    datetime: "2018-06-15T07:00:00.000-05:00",

Can you please explain to which time zone this time refers to ? 
Timing as per FIFA site is just mentioned as Local Time for the above time.
Again Russia has multiple zones. 
Unable to derive the time for a country-specific time, for e.g India in my case.

**Comment by estiens** [comment]
Yeah, the server is set in UTC-5, so you'll need to convert to whatever you are in, for example, if you are in UTC+5 you would need to add 10 hours to that -- most languages can import that string directly to a UTC datetime which you can then output in a local timezone.

I can also switch to display the times in pure UTC in the API
